GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -168,12 +168,13 @@ public FileStatusDiffCalculator(Func<GitModule> getModule)
                 statuses: commonBaseToAandB));
 
             // Add rangeDiff as a separate group (range is not the same as diff with artificial commits)
-            List<GitItemStatus> statuses = new List<GitItemStatus> { new GitItemStatus { Name = Strings.DiffRange, IsRangeDiff = true } };
-
-            var desc = Strings.DiffRange + ": " + GetDescriptionForRevision(describeRevision, firstRevHead) + " -> " +
-                       GetDescriptionForRevision(describeRevision, selectedRevHead);
+            var statuses = new List<GitItemStatus> { new GitItemStatus { Name = Strings.DiffRange, IsRangeDiff = true } };
             var first = firstRev.ObjectId == firstRevHead ? firstRev : new GitRevision(firstRevHead);
             var selected = selectedRev.ObjectId == selectedRevHead ? selectedRev : new GitRevision(selectedRevHead);
+            var (baseToFirstCount, baseToSecondCount) = module.GetCommitRangeDiffCount(first.ObjectId, selected.ObjectId);
```

Comment:
There are some more methods needed for that to happen.
Also, I would prefer to not share this with plugins. If so, the methods should change to strings instead of ObjectId (as there may be situations where treeish identifiers like HEAD could be used.)

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'IGitModule' does not contain a definition for 'GetTreeFiles' and no accessible extension method 'GetTreeFiles' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	40	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetCombinedDiffFileList' and no accessible extension method 'GetCombinedDiffFileList' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	61	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetCommitDiffCount' and no accessible extension method 'GetCommitDiffCount' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	191	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetCommitDiffCount' and no accessible extension method 'GetCommitDiffCount' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	192	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetCommitRangeDiffCount' and no accessible extension method 'GetCommitRangeDiffCount' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	173	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetDiffFilesWithSubmodulesStatus' and no accessible extension method 'GetDiffFilesWithSubmodulesStatus' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	54	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetDiffFilesWithSubmodulesStatus' and no accessible extension method 'GetDiffFilesWithSubmodulesStatus' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	87	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetDiffFilesWithSubmodulesStatus' and no accessible extension method 'GetDiffFilesWithSubmodulesStatus' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	138	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetDiffFilesWithSubmodulesStatus' and no accessible extension method 'GetDiffFilesWithSubmodulesStatus' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	147	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetDiffFilesWithSubmodulesStatus' and no accessible extension method 'GetDiffFilesWithSubmodulesStatus' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	148	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetMergeBase' and no accessible extension method 'GetMergeBase' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	103	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetMergeBase' and no accessible extension method 'GetMergeBase' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	108	Active
Error	CS1061	'IGitModule' does not contain a definition for 'GetMergeBase' and no accessible extension method 'GetMergeBase' accepting a first argument of type 'IGitModule' could be found (are you missing a using directive or an assembly reference?)	GitUI	F:\dev\gc\gitextensions\GitUI\UserControls\FileStatusDiffCalculator.cs	111	Active
```

Reply:
This is something we'll need to fix going forward.